addons/website/static/tests/tours/website_click_tests.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module **/
+
+import wTourUtils from 'website.tour_utils';
+
+const cover = {
+    id: 's_cover',
+    name: 'Cover',
+};
+
+wTourUtils.registerWebsitePreviewTour('website_click_tour', {
+    test: true,
+    url: '/',
+}, [
+    {
+        content: "trigger a page navigation",
+        trigger: 'iframe a[href="/contactus"]',
+    },
+    {
+        content: "wait for the page to be loaded",
+        trigger: 'iframe a.nav-link.active[href="/contactus"]',
+        run: () => null, // it's a check
+    },
+    wTourUtils.clickOnEdit(),
+    {
+        content: "wait for edition to be started",
+        trigger: 'iframe body.editor_enable',
+        run: () => null, // it's a check
+    },
+    {
+        content: "click on a link that would trigger navigation",
+        trigger: 'iframe a[href="/"]',
+    },
```

Comment:
@detrouxdev thanks for the test

@xO-Tx I though I would share it, even if it might seems obvious as I am not sure it actually is: when reviewing PRs and there is test involved, you should ensure that the test is failing without the fix and succeeding with the fix, it will ensure the test actually tests something.

The developer should already have made sure of that but it's still a triple check for the reviewer to do :+1: